templates/web/pages/export_products/export_products.tt.js

```diff
@@ -0,0 +1,73 @@
+
+var poll_n1 = 0;
+var timeout1 = 5000;
+var job_info_state1;
+
+var poll_n2 = 0;
+var timeout2 = 5000;
+var job_info_state2;
+
+var poll_n3 = 0;
+var timeout3 = 5000;
+var job_info_state3;
+
+var minion_status = {
+	"inactive" : [% lang("minion_status_inactive") %],
+	"active" : [% lang("minion_status_active") %],
+	"finished" : [% lang("minion_status_finished") %],
+	"failed" : [% lang("minion_status_failed") %]
+};
+
+(function poll1() {
+    \$.ajax({
```

Comment:
I think \$ should be $.

Have you tried running the code to check that it still works as expected?